server/src/main/resources/db/dev/R__Load_testing_data.sql

```diff
@@ -822,11 +822,6 @@ insert into role_permissions
 values
     ('e8a4fff8-e984-4e59-be84-a713c9fa8d23', 'CAN_ADMINISTER_SETTINGS');
 
-insert into role_permissions
-(roleid, permission)
-values
```

Comment:
This was a duplicate section causing the database to fail. Not sure how this got in. Likely a mistake on develop

Reply:
Pretty sure this was my fault from a merge I did recently. Good catch. I think Tim just pushed a change for this to develop as well.

Reply:
Ah! Was just relieved it wasn't another of my own foibles.

Reply:
Nope just my own haha.